src/otel/OTelGroupCallMembership.ts

```diff
@@ -74,7 +74,7 @@ function flattenVoipEventRecursive(
     );
 
   for (const [k, v] of Object.entries(obj)) {
-    if (["string", "number", "boolean"].includes(typeof v)) {
+    if (["string", "number", "boolean"].includes(typeof v) || v === null) {
       flatObject[prefix + k] = v;
     } else if (typeof v === "object") {
       flattenVoipEventRecursive(
```

Comment:
FYI: We have a new [`flattenVoipEventRecursive`](https://github.com/vector-im/element-call/blob/a2b3e098b6c650f05e6db495d0ebabe17a2567d3/src/otel/ObjectFlattener.ts#L64)   which is an extended version of this here. The new one can also handle Maps.